### DISCUSSION TITLE
SD seems to run really slowly through the interface...

### DISCUSSION THREAD

**Discussion by helmutkravitz** [hlky/stable-diffusion-webui#394] [Q&A]
I assume I'm doing something incorrectly to get this result, but I'd been using another Colab notebook to run SD, and though I vastly prefer the interface and features you're offering here, I'm finding my renders are taking about 5x times as long with the same model settings as they were on the previous notebook. Any recommendations for common defaults or settings that might dramatically slow the model? I'm using a Colab Pro account fwiw.

Thanks, appreciate any thoughts.

**Comment by Bebra2007** [comment]
Maybe the problem is that Colab machines have better harware than your PC. If i remember correctly colab has 16 gb gpus
